.vsts-ci.yml

```diff
@@ -182,7 +182,7 @@ steps:
 - task: PowerShell@2
   displayName: Publish Drop - VSSetup
   inputs:
-    arguments: '-Source Binaries\VSSetup\$(BuildConfiguration)\Insertion -Target $(DropRoot)\Roslyn-Signed\$(Build.SourceBranchName)\$(BuildConfiguration)\$(Build.BuildNumber)\Insertion -ParallelCount 8'
+    arguments: '-Source Binaries\VSSetup\$(BuildConfiguration) -Target $(DropRoot)\Roslyn-Signed\$(Build.SourceBranchName)\$(BuildConfiguration)\$(Build.BuildNumber)\VSSetup -ParallelCount 8'
```

Comment:
Why is the target path changing like it is? I could have imagined you were trying to keep it in sync so the setup stuff lives in the same place, but this isn't doing that. (I think?)

Reply:
`Insertion` is a subdirectory of `VSSetup`, so this uploads the entire `VSSetup` directory instead of just it's `Insertion` subdirectory.

This is just a temporary change anyways. We'll remove all this once we move away from cpvsdrops.

Reply:
@jasonmalinowski Ping

Reply:
Does this need other changes though on the insertion tool? I would have assumed it was looking for an "Insertion" folder but now it'll be "VSSetup\Insertion" on cpvsbuild?

Reply:
The insertion tool does not consume these files.